fdbclient/TestKnobCollection.h

```diff
@@ -26,19 +26,31 @@
 class TestKnobs : public KnobsImpl<TestKnobs> {
 public:
 	TestKnobs();
+
+	// Nonatomic test knobs
 	int64_t TEST_LONG;
 	int TEST_INT;
 	double TEST_DOUBLE;
 	bool TEST_BOOL;
 	std::string TEST_STRING;
+
+	// Atomic test knobs
+	int64_t TEST_ATOMIC_LONG;
+	int TEST_ATOMIC_INT;
+	double TEST_ATOMIC_DOUBLE;
+	bool TEST_ATOMIC_BOOL;
+	std::string TEST_ATOMIC_STRING;
+
 	bool operator==(TestKnobs const&) const;
 	bool operator!=(TestKnobs const&) const;
 	void initialize();
+
+	bool isAtomic(std::string const& knob) const { return knob.find("atomic") != std::string::npos; }
```

Comment:
Why does `isAtomic` need to be defined here? Doesn't the `Knobs::isAtomic` implementation work?

Reply:
It doesn't, don't know why I added it 😅